<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataRecorderWidget</class>
 <widget class="QFrame" name="DataRecorderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="dataRecSettingsGroupBox">
     <property name="title">
      <string>Data Recorder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="channel1Label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Channel 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recsrc1ComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="channekl2Label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left: 6px</string>
        </property>
        <property name="text">
         <string>Channel 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recsrc2ComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="recDurationLabel">
        <property name="styleSheet">
         <string notr="true">margin-left: 6px</string>
        </property>
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="recDurationSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="samplingPeriodLabel">
        <property name="styleSheet">
         <string notr="true">margin-left: 6px</string>
        </property>
        <property name="text">
         <string>Sampling Period</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="samplePeriodSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <double>16777215.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="historyLabel">
        <property name="text">
         <string>Keep History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="historyCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toggleSwitch" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearPlotButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clear Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MplWidget" name="mplWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: lightgrey</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">pysoworks.mplcanvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
